sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -717,6 +756,63 @@ export class KeyClient {
       return { value: result.value! };
     });
   }
+
+  /**
+   * Gets the rotation policy of a Key Vault Key.
+   *
+   * Example usage:
+   * ```ts
+   * let client = new KeyClient(vaultUrl, credentials);
+   * await client.setKeyRotationPolicy("MyKey", myPolicy);
+   * let result = await client.getKeyRotationPolicy("myKey");
+   * ```
+   *
+   * @param name - The name of the key.
+   * @param options - The optional parameters.
+   */
+  public getKeyRotationPolicy(
+    name: string,
+    options: GetRotationPolicyOptions = {}
+  ): Promise<KeyRotationPolicy | undefined> {
+    return withTrace("getRotationPolicy", options, async () => {
+      const policy = await this.client.getKeyRotationPolicy(this.vaultUrl, name);
+      if (policy.id) {
```

Comment:
If a key has no policy set, it will still return a response with { id: undefined, attributes: null, lifetimeActions: [] } which would be IMO confusing. 

Do we have precedent or guidelines for what to do with a "empty" response - as in it returns an object but it's empty?